### PR TITLE
Skip unit tests for PRs containing only integration test changes

### LINKED ISSUE #9772: Don't run unit tests for changes involving just integration tests
Today, if you are making a quick change to disable a flaky integration test, the PR sits on a Travis queue for about 20 minutes in order to run unit tests (that don't really exercise the change) before allowing you to merge.

For PRs that contain *only* integration test changes, we should still do a proper build and run presubmit and other tests, but we should skip unit tests. This will save a considerable amount of time on the Travis queue.

### PR DESCRIPTION
**Pull request by rsimha-amp**
Since we run integration tests only on master, this PR will speed up Travis PR builds if all you are doing is changing / disabling / enabling an integration test.

Fixes #9772

### CODE REVIEW

File: build-system/pr-check.js [hunk 1]
```diff
@@ -155,6 +156,15 @@ function isDocFile(filePath) {
 }
 
 /**
+ * Determines if the given file is an integration test.
+ * @param {string} filePath
+ * @return {boolean}
+ */
+function isIntegrationTest(filePath) {
+  if (filePath.startsWith('test/integration/')) return true;
```
Comment (erwinmombay) [inline build-system/pr-check.js]: no need for if statement

Comment (rsimha-amp) [inline build-system/pr-check.js]: Coming up in a different PR.

**Comment by rsimha-amp** [comment]
This was a success. See https://travis-ci.org/ampproject/amphtml/jobs/240510675. Travis runs for PRs containing just integration test changes reduced by half, to about 9 mins.

**Pull request merged**